fix Chain issues: Incorrect order, Extra certs

kaefert opened the pull request:
Using your script on my Qnap TS-469L running firmware version 4.2.1 will make the SSL test service found @
https://www.ssllabs.com/ssltest/analyze.html
report Chain issues: Incorrect order, Extra certs

My proposed change will fix this and change the relevant line in the report to:
Chain issues: None

szech commented:
thanks! i expect there are many things that could be better in the script but it has been "good enough" for now :)

kaefert commented:
hey there szech! Yes, the expiration check also does not work, since the used path `letsencrypt/signed.crt` is wrong. That should be `letsencrypt/live/$DOMAIN/cert.pem`

I've reworked the script and added lots of echo comments to understand it better, see https://github.com/kaefert/qnap-letsencrypt/blob/master/renew_certificate.sh

Big thanks for creating this script and by doing so showing me how to most conveniently getting letsencrypt auto renewal on my Qnap NAS! :)

szech commented:
I'm happy you find it useful :)
All the original heavy lifting was done by the other ppl i mention in the
FAQ, so they deserve a lot of credit.

# 

  http://twitter.com/captainszechwan
https://www.songkick.com/users/captainszechwan
http://www.last.fm/user/CaptainSzechwan

On 30 July 2016 at 08:34, Thomas notifications@github.com wrote:

> hey there szech! Yes, the expiration check also does not work, since the
> used path letsencrypt/signed.crt is wrong. That should be
> letsencrypt/live/$DOMAIN/cert.pem
> 
> I've reworked the script and added lots of echo comments to understand it
> better, see
> https://github.com/kaefert/qnap-letsencrypt/blob/master/renew_certificate.sh
> 
> Big thanks for creating this script and by doing so showing me how to most
> conveniently getting letsencrypt auto renewal on my Qnap NAS! :)
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> https://github.com/szech/qnap-letsencrypt/pull/1#issuecomment-236350987,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AGTwOoYp6QyuLWwRJssQ8-UIkusMfAZMks5qav6CgaJpZM4JWYb2
> .
